src/web/app/components/question-submission-form/question-submission-form.component.html

```diff
@@ -183,8 +183,12 @@
       <div class="col-12 text-center">
         <button id="btn-submit-qn-{{ model.questionNumber }}" type="submit" class="btn btn-success"
                 ngbTooltip="You can save your responses for this question at any time and come back later to continue."
-                (click)="saveFeedbackResponses()" [disabled]="isSavingResponses || isSubmissionDisabled"
-        ><tm-ajax-loading *ngIf="isSavingResponses"></tm-ajax-loading>Submit Responses</button>
+                (click)="saveFeedbackResponses()" [disabled]="isSavingResponses || isSubmissionDisabled">
+          <tm-ajax-loading *ngIf="isSavingResponses"></tm-ajax-loading>
+          <span>
+            {{ "Submit Response for Question " + model.questionNumber }}
```

Comment:
Just checked on my end and it seems that this still adds an additional space. Its ok for you to add the opening and closing tag of the span element on the same line e.g <span>xxx</span>

![image](https://user-images.githubusercontent.com/25302138/173237025-cc180784-6635-4123-862a-47cc9ed48ace.png)

Reply:
Sorry about that. Should be fixed now. 

![image](https://user-images.githubusercontent.com/53614454/173242526-b58a1212-1820-4617-96e1-63ea5ec83f45.png)